README.md

```diff
@@ -13,46 +13,111 @@ The image does not contain an entry point, because the build runner executes the
 
 ``` yaml
 trigger:
-- master
+- azure-pipelines
+
+pool:
+  vmImage: 'ubuntu-latest'
 
 resources:
   containers:
   - container: platformio
     image: infinitecoding/platformio-for-ci:latest
-    endpoint: dockerhub-infinite-coding
+    endpoint: personal-docker-hub-connection
+
+container: platformio
+steps:
+
+- script: platformio run -d ./example/
+  displayName: 'Build firmware'
+
+- task: CopyFiles@2
+  inputs:
+    SourceFolder: '$(Build.SourcesDirectory)/example/.pio/build/esp32dev/'
+    Contents: 'firmware.bin'
+    TargetFolder: '$(Build.ArtifactStagingDirectory)'
+
+- task: PublishBuildArtifacts@1
+  inputs:
+    ArtifactName: 'Firmware $(Build.BuildNumber)'
+    PathtoPublish: $(Build.ArtifactStagingDirectory)
+    publishLocation: Container
+    TargetPath: .
+```
+
+# Github Actions example
+
+The following build script builds the project ESP32 firmware in the [example](example) folder withing the *infinitecoding/platformio-for-ci* Docker container with github actions.
+
+![Example-CI](https://github.com/Andreas-Huber/platformio-docker-for-ci/workflows/Example-CI/badge.svg?branch=github-actions)
+
+**[.github\workflows\platformio-example.yml](.github\workflows\platformio-example.yml)**
+
+``` yaml
+name: Example-CI
+
+on:
+  push:
+    branches: [ github-actions ]
 
 jobs:
-- job: esp32_platformio
-  displayName: "PlatformIO build"
-  container: platformio
-  pool:
-    vmImage: ubuntu-18.04
-  steps:
-    - script: |
-        rm -fdr bin
-        mkdir bin
-      displayName: 'Prepare output directories'      
-    - script: platformio ci --build-dir="./bin" --keep-build-dir --project-conf=platformio.ini ./helloworld/
-      displayName: 'Build firmware'
-    - task: CopyFiles@2
-      inputs:
-        SourceFolder: $(Build.SourcesDirectory)/bin/.pio/build/esp32dev/
-        Contents: 'firmware.bin'
-        TargetFolder: $(Build.ArtifactStagingDirectory)
-    - task: PublishBuildArtifacts@1
-      inputs:
-        ArtifactName: 'Firmware $(Build.BuildNumber)'
-        PathtoPublish: $(Build.ArtifactStagingDirectory)
-        publishLocation: Container
-        TargetPath: .
-```
\ No newline at end of file
+  platformio-build:
+    runs-on: ubuntu-latest
+    container: infinitecoding/platformio-for-ci:latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build firmware
+        run: platformio run -d ./example/
+
+      - name: Upload firware artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: firmware.elf
+          path: example/.pio/build/esp32dev/firmware.elf
+```
+
+# Run the container locally
+
+To test if the PlatformIO build works with the container, you can also run it locally.
+
+## Linux & MAC
+To build the project in the example folder on a UNIX system run the following commands.
+Or run the [runBuildInDocker.sh](example/runBuildInDocker.sh) bash script.
+
+``` bash
+# Set the example project folder as working directory
+cd example

```

Comment:
do you really have to change directory? you could just append the folder in the docker run i guess

Reply:
Of course, just thought this is more clear that way that the actual project is in the example folder. Because it is within this GitHub repository. Usually, you would run the command in the root of your project.